cypress/support/commands.ts

```diff
@@ -398,6 +399,20 @@ export const createMapVariable = (
   })
 }
 
+export const createMapVariableFromFix = (
```

Comment:
```suggestion
export const createMapVariableFromFixture = (
```

`fromFix` makes me think bugfix, not fixture.

Reply:
refactored